.github/ISSUE_TEMPLATE/api-suggestion.md

```diff
@@ -0,0 +1,54 @@
+---
+name: API proposal
+about: Propose a change to the public API surface
+title: ''
+labels: [Feature Request, Concept-API]
+assignees: ''
+
+---
+
+## Background and Motivation
+
+<!--
+We welcome API proposals! We have a process to evaluate the value and shape of new API. There is an overview of our process [here](https://github.com/dotnet/roslyn/blob/main/docs/contrbuting/API%20Review%20Process.md). This template will help us gather the information we need to start the review process.
+First, please describe the purpose and value of the new API here.
+-->
+
+## Proposed API
+
+<!--
+Please provide the specific public API signature diff that you are proposing. For example:
```

Comment:
Can we give a reassurance that you can give your best effort or guess at the best API for what you're asking, and the proposal won't be pigeonholed because of the specificity resulting from following this section?

Following from @YairHalberstadt's comment:

> What should the process be for someone who wants an API to be added but isn't yet clear on the exact shape it would take?

Reply:
I like the word 'sketch' from the contributing document below.

Reply:
I've reworded this a bit, including the word `sketch` to try and make it a bit less onerous. However, ultimately I don't think we intend to try and drive design off a statement of "I want an API to do X". It's certainly possible that someone on the team could see such a statement, think that it's a good idea, and come up with a proposal that would fit the "sketch" requirement, but given the amount of features that we could be implementing I don't want to encourage drive-by sentences like that.